<!DOCTYPE html>
<html lang="en"><head>
    <link rel="apple-touch-icon" sizes="180x180" href="https://dbrg77.github.io/SUSTech-BIO210/apple-touch-icon.png">
    <link rel="icon" type="image/png" sizes="32x32" href="https://dbrg77.github.io/SUSTech-BIO210/favicon-32x32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="https://dbrg77.github.io/SUSTech-BIO210/favicon-16x16.png">
    <link rel="manifest" href="https://dbrg77.github.io/SUSTech-BIO210/site.webmanifest">
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <style>
        :root {
            --accent-color: #FF4D4D;
        }
    </style>

    
    
    
    
    
    

    
    <title>Lecture 22 Confidence Interval For The Proportion</title>
    <meta name="description" content="by School of Life Sciences at SUSTech">
    <meta name="keywords" content='blog, gokarna, hugo, data science, biology, genomics'>

    <meta property="og:url" content="https://dbrg77.github.io/SUSTech-BIO210/posts/lecture-22/">
    <meta property="og:type" content="website">
    <meta property="og:title" content="Lecture 22 Confidence Interval For The Proportion">
    <meta property="og:description" content="by School of Life Sciences at SUSTech">
    <meta property="og:image" content="/images/avatar.png">

    <meta name="twitter:card" content="summary_large_image">
    <meta name="twitter:title" content="Lecture 22 Confidence Interval For The Proportion">
    <meta name="twitter:description" content="by School of Life Sciences at SUSTech">
    <meta property="twitter:domain" content="https://dbrg77.github.io/SUSTech-BIO210/posts/lecture-22/">
    <meta property="twitter:url" content="https://dbrg77.github.io/SUSTech-BIO210/posts/lecture-22/">
    <meta name="twitter:image" content="/images/avatar.png">

    
    <link rel="canonical" href="https://dbrg77.github.io/SUSTech-BIO210/posts/lecture-22/" />

    <link rel="stylesheet" type="text/css" href="https://dbrg77.github.io/SUSTech-BIO210/css/normalize.min.css" media="print" onload="this.media='all'">
    <link rel="stylesheet" type="text/css" href="https://dbrg77.github.io/SUSTech-BIO210/css/main.css">
    <link disabled id="dark-theme" rel="stylesheet" href="https://dbrg77.github.io/SUSTech-BIO210/css/dark.css">

    <script src="https://dbrg77.github.io/SUSTech-BIO210/js/svg-injector.min.js"></script>
    <script src="https://dbrg77.github.io/SUSTech-BIO210/js/feather-icons.min.js"></script>
    <script src="https://dbrg77.github.io/SUSTech-BIO210/js/main.js"></script>

    
    
        <!-- KaTeX -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.13.16/dist/katex.min.css" integrity="sha384-6LkG2wmY8FK9E0vU9OOr8UvLwsaqUg9SETfpq4uTCN1agNe8HRdE9ABlk+fVx6gZ" crossorigin="anonymous">
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.13.16/dist/katex.min.js" integrity="sha384-31El76TwmbHj4rF9DyLsygbq6xoIobG0W+jqXim+a3dU9W53tdH3A/ngRPxOzzaB" crossorigin="anonymous"></script>
    <script defer src="https://cdn.jsdelivr.net/npm/katex@0.13.16/dist/contrib/auto-render.min.js" integrity="sha384-vZTG03m+2yp6N6BNi5iM4rW4oIwk5DfcNdFfxkk9ZWpDriOkXX8voJBFrAO7MpVl" crossorigin="anonymous"
        onload="renderMathInElement(document.body);"></script>
    <script>
        document.addEventListener("DOMContentLoaded", function() {
            renderMathInElement(document.body, {
                delimiters: [
                    {left: "$$", right: "$$", display: true},
                    {left: "$", right: "$", display: false}
                ]
            });
        });
    </script>
  
    
</head>
<body>
        <script type="text/javascript">
            
            setThemeByUserPref();
        </script><header class="header">
    <nav class="header-nav">

        
        <div class="avatar">
            <a href="https://dbrg77.github.io/SUSTech-BIO210/">
                <img src="https://dbrg77.github.io/SUSTech-BIO210//images/avatar.png" alt="avatar" />
            </a>
        </div>
        

        <div class="nav-title">
            <a class="nav-brand" href="https://dbrg77.github.io/SUSTech-BIO210/">SUSTech BIO210 Biostatistics</a>
        </div>

        <div class="nav-links">
            
            <div class="nav-link">
                <a href="https://dbrg77.github.io/SUSTech-BIO210"><span data-feather='home'></span> Home </a>
            </div>
            
            <div class="nav-link">
                <a href="https://dbrg77.github.io/SUSTech-BIO210/posts/"><span data-feather='book-open'></span> Posts </a>
            </div>
            
            <div class="nav-link">
                <a href="https://dbrg77.github.io/SUSTech-BIO210/course/"><span data-feather='list'></span> Content Index </a>
            </div>
            
            <div class="nav-link">
                <a href="https://dbrg77.github.io/SUSTech-BIO210/about/"><span data-feather='file-text'></span> About </a>
            </div>
            
            <div class="nav-link">
                <a href="https://github.com/dbrg77/SUSTech-BIO210"><span data-feather='github'></span>  </a>
            </div>
            

            <span class="nav-icons-divider"></span>
            <div class="nav-link dark-theme-toggle">
                <span id="dark-theme-toggle-screen-reader-target" class="sr-only"></span>
                <a>
                    <span id="theme-toggle-icon" data-feather="moon"></span>
                </a>
            </div>

            <div class="nav-link" id="hamburger-menu-toggle">
                <span id="hamburger-menu-toggle-screen-reader-target" class="sr-only">menu</span>
                <a>
                    <span data-feather="menu"></span>
                </a>
            </div>

            
            <ul class="nav-hamburger-list visibility-hidden">
                
                <li class="nav-item">
                    <a href="https://dbrg77.github.io/SUSTech-BIO210"><span data-feather='home'></span> Home </a>
                </li>
                
                <li class="nav-item">
                    <a href="https://dbrg77.github.io/SUSTech-BIO210/posts/"><span data-feather='book-open'></span> Posts </a>
                </li>
                
                <li class="nav-item">
                    <a href="https://dbrg77.github.io/SUSTech-BIO210/course/"><span data-feather='list'></span> Content Index </a>
                </li>
                
                <li class="nav-item">
                    <a href="https://dbrg77.github.io/SUSTech-BIO210/about/"><span data-feather='file-text'></span> About </a>
                </li>
                
                <li class="nav-item">
                    <a href="https://github.com/dbrg77/SUSTech-BIO210"><span data-feather='github'></span>  </a>
                </li>
                
                <li class="nav-item dark-theme-toggle">
                    <span id="dark-theme-toggle-screen-reader-target" class="sr-only">theme</span>
                    <a>
                        <span id="theme-toggle-icon" data-feather="moon"></span>
                    </a>
                </li>
            </ul>

        </div>
    </nav>
</header>
<main id="content">
    <div class="post container">
    <div class="post-header-section">
        <h1>Lecture 22 Confidence Interval For The Proportion</h1>
        <small role="doc-subtitle"></small>
        <p class="post-date">
            April 4, 2023
        </p>

        <ul class="post-tags">
        
        </ul>
    </div>

    <div class="post-content">
        <p>
            <p>Now that we have figured out the sampling distribution of the sample proportion, we can make interval estimation and construct confidence intervals for the <strong>sample proportion</strong>. Before we do that, let&rsquo;s use an example to look the distribution of $P$ one more time.</p>
<h2 id="a-simplified-version-of-gene-expression">A Simplified Version of Gene Expression</h2>
<p>As we know, <strong>DNA</strong> is our genetic material. In order for a gene to function, the gene needs to be &ldquo;converted&rdquo; into proteins<sup id="fnref:1"><a href="#fn:1" class="footnote-ref" role="doc-noteref">1</a></sup>. This process is describe by the <strong>central dogma of molecular biology</strong> that we learnt in high school. Here is the draft version of it from <strong>Francis Crick&rsquo;s</strong> conference notes in a symposium held at London back in 1957, one year before the central dogma was officially published:</p>
<figure><img src="https://dbrg77.github.io/SUSTech-BIO210/images/post_figures/central_dogma.jpg" width="700px"/>
</figure>

<blockquote>
<p>Credit: &ldquo;Ideas on protein synthesis (Oct. 1956)&rdquo;. <a href="https://wellcomecollection.org/works/xmscu3g4">Wellcome Collection</a>. <a href="http://rightsstatements.org/vocab/InC/1.0/">In copyright</a></p>
</blockquote>
<p>The first step is the conversion into RNA using DNA as the template, a procedure called <strong>transcription</strong>. In modern days, we have sequencing technologies to quantify the number of RNA molecules of each gene in a particular sample (see <strong>Lecture 1</strong>). Getting those numbers is very important, because they can tell us many things about the sample we study, such as the status and the identity. Different genes may have different numbers of RNA molecules in a sample. For example, the gene <em>GAPDH</em> encodes an enzyme that is important for glycolysis. The cells need a lot of it, so it has many RNA molecules to be translated into proteins. Other genes may not have that many RNA molecules, not because they are not important,but for many complicated reasons which are left for you to explore in your molecular and cell biology class.</p>
<p>There are over 20,000 genes. Each time we perform an experiment, we get certain number of molecules from a sample. Those molecules basically come from those 20,000 genes, so we have 20,000 integer values for each experiment, representing <strong>molecular counts</strong> for those genes. However, experiments are ever perfect. We never get the same amount of molecules even for the same sample. Let&rsquo;s say, we have three different experiments. In the first one, we are able to obtain 20 million molecules; in the second experiment, we capture 40 million molecules and the third 30 million molecules. Once we are done, we observe that the number of molecules of <em>GAPDH</em> in those three experiments are $1990$, $4020$ and $3005$, respectively. Can we say that the <strong>true values</strong> of <em>GAPDH</em> in those three experiments are different? Probably not. The differences might be caused by the total number of molecules we get per experiment. Remember the total number of molecules are the results of 20,000 genes. Therefore, it is the relative quantity of the RNA of the gene that is of our interest, that is, the <strong>sample proportion</strong> we talked about. We call it the <strong>gene expression level</strong>.</p>
<p>Now let&rsquo;s see a different gene called <em>PLK1</em>, which is an essential genes for the cell cycle progression. Its gene expression level is not as high as <em>GAPDH</em>, and say we know its true gene expression level as:</p>
<p>$$\pi = 0.000001126088083$$</p>
<p>Let&rsquo;s think about this number for a moment before we move on. This might look like a really <em>really</em> tiny fraction, but if you think about it a bit more, it actually makes sense. When we talk about the proportion in terms of the <strong>fraction</strong> or the <strong>percentage</strong>, the total sum is $1$ or $100%$, respectively. That comes from over 20,000 genes, so eventually each gene only takes a very small fraction or percentage of the total. To make that number look a bit prettier, we do not talk about percentage <strong>(per $\boldsymbol{100}$)</strong> when dealing with gene expression levels. We use <strong>per million ($\boldsymbol{10^6}$)</strong>, which is still just a proportion at a different scale.</p>
<p>Now we ask the following question: what is the sampling distribution of the sample proportion of <em>PLK1</em> if we draw one million $(10^6)$ molecules from the population? We use what we learnt in the last lecture and expect the proportion should follow:</p>
<p>$$P_{PLK1} \ \dot\sim \ \mathcal{N} \left(\mu_{P} = \pi = 1.126 \times 10^{-6}, \sigma_{P}^2 = \cfrac{\pi(1-\pi)}{n} = 1.126 \times 10^{-12} \right)$$</p>
<p>We can do some simulations on the computer to see if our expectation is correct. The following is some very crude <code>python</code> codes for this purpose, but it works and the speed is acceptable:</p>
<div class="highlight"><pre tabindex="0" style="color:#abb2bf;background-color:#282c34;-moz-tab-size:4;-o-tab-size:4;tab-size:4;"><code class="language-python" data-lang="python"><span style="display:flex;"><span><span style="color:#7f848e"># import some helper modules</span>
</span></span><span style="display:flex;"><span><span style="color:#c678dd">import</span> <span style="color:#e06c75">numpy</span> <span style="color:#c678dd">as</span> <span style="color:#e06c75">np</span>
</span></span><span style="display:flex;"><span><span style="color:#c678dd">from</span> <span style="color:#e06c75">collections</span> <span style="color:#c678dd">import</span> <span style="color:#e06c75">Counter</span>
</span></span><span style="display:flex;"><span>
</span></span><span style="display:flex;"><span><span style="color:#7f848e"># set the seed</span>
</span></span><span style="display:flex;"><span><span style="color:#e06c75">np</span><span style="color:#56b6c2">.</span><span style="color:#e06c75">random</span><span style="color:#56b6c2">.</span><span style="color:#e06c75">seed</span>(<span style="color:#d19a66">42</span>)
</span></span><span style="display:flex;"><span>
</span></span><span style="display:flex;"><span><span style="color:#7f848e"># the expression level of PLK1 is 0.000001126088083</span>
</span></span><span style="display:flex;"><span><span style="color:#7f848e"># which is roughly 100 per 88803000</span>
</span></span><span style="display:flex;"><span><span style="color:#7f848e"># so we create a array with 88802900 zeroes and 100 ones</span>
</span></span><span style="display:flex;"><span><span style="color:#e06c75">molecules</span> <span style="color:#56b6c2">=</span> <span style="color:#e06c75">np</span><span style="color:#56b6c2">.</span><span style="color:#e06c75">zeros</span>((<span style="color:#d19a66">88803000</span>,))
</span></span><span style="display:flex;"><span><span style="color:#e06c75">molecules</span>[:<span style="color:#d19a66">100</span>] <span style="color:#56b6c2">=</span> <span style="color:#d19a66">1</span>
</span></span><span style="display:flex;"><span>
</span></span><span style="display:flex;"><span><span style="color:#7f848e"># then we take 1000000 molecules from that</span>
</span></span><span style="display:flex;"><span><span style="color:#7f848e"># check how many 1s (PLK1 molecules) there</span>
</span></span><span style="display:flex;"><span><span style="color:#7f848e"># record the number</span>
</span></span><span style="display:flex;"><span><span style="color:#7f848e"># repeat for 1000 times</span>
</span></span><span style="display:flex;"><span><span style="color:#e06c75">results</span> <span style="color:#56b6c2">=</span> [] <span style="color:#7f848e"># hold the number of PLK1 per experiment</span>
</span></span><span style="display:flex;"><span><span style="color:#7f848e">## the following loop takes a few seconds</span>
</span></span><span style="display:flex;"><span><span style="color:#c678dd">for</span> <span style="color:#e06c75">_</span> <span style="color:#56b6c2">in</span> <span style="color:#e5c07b">range</span>(<span style="color:#d19a66">1000</span>):
</span></span><span style="display:flex;"><span>    <span style="color:#e06c75">plk1</span> <span style="color:#56b6c2">=</span> <span style="color:#e06c75">np</span><span style="color:#56b6c2">.</span><span style="color:#e06c75">random</span><span style="color:#56b6c2">.</span><span style="color:#e06c75">choice</span>(<span style="color:#e06c75">molecules</span>, <span style="color:#d19a66">1000000</span>)<span style="color:#56b6c2">.</span><span style="color:#e06c75">sum</span>() <span style="color:#7f848e"># take 1e6 molecules, check how many 1s (PLK1)</span>
</span></span><span style="display:flex;"><span>    <span style="color:#e06c75">results</span><span style="color:#56b6c2">.</span><span style="color:#e06c75">append</span>(<span style="color:#e06c75">plk1</span>)                              <span style="color:#7f848e"># record the number</span>
</span></span><span style="display:flex;"><span>
</span></span><span style="display:flex;"><span><span style="color:#7f848e"># now look at the results</span>
</span></span><span style="display:flex;"><span><span style="color:#e06c75">Counter</span>(<span style="color:#e06c75">results</span>)
</span></span></code></pre></div><p>If you plot the results as PMF, you will see:</p>
<figure><img src="https://dbrg77.github.io/SUSTech-BIO210/images/post_figures/plk1_sim.png" width="500px"/>
</figure>

<p>That looks like a positively skewed distribution. It will probably not be approximated by a normal distribution well, since normal distributions are symmetrical. Indeed, if we plot the normal distribution $\mathcal{N}\left(1.126 \textmd{ per million}, 1.126 \textmd{ per million}^2\right)$ on top of it, we can see:</p>
<figure><img src="https://dbrg77.github.io/SUSTech-BIO210/images/post_figures/plk1_sim_norm.png" width="500px"/>
</figure>

<p>There are two things that we noticed from the above picture. The first one is that the centre of the gravity of the simulation data (red bars) lies between $0$ and $1$, but the theoretical normal distribution (the grey line) is centred on $1.126$. The second thing is that the simulation data does not take the negative values, but there is still a lot of data in the normal distribution at the negative side of the axis.</p>
<p>The red bars look a lot like a Poisson distribution. Indeed, if we put on top of a Poisson PMF with $\lambda = 1.126 \textmd{ per million}$, we have:</p>
<figure><img src="https://dbrg77.github.io/SUSTech-BIO210/images/post_figures/plk1_sim_pois.png" width="500px"/>
</figure>

<h2 id="conditions-of-the-normal-approximation">Conditions of The Normal Approximation</h2>
<p>We were told that as long as the sample size is large enough, the sample proportion (the sample mean of the indicator variables) should follows a normal distribution, regardless of the distribution of the original population. Any reasonable person would think 1 million should be a sample size that is <strong>large enough</strong>. However, it seems the normal distribution does not work here. So what is going on here?</p>
<p>As we discussed in the last lecture, the distribution of the <em>PLK1</em> gene expression in our experiment is essentially a binomial distribution with a really big $n$ and a really tiny $p$. It turns out that when the product of $n \cdot p$ small, the binomial is well approximated by a Poisson distribution with $\lambda = np$. When $n \cdot p$ is not small, the binomial can be approximated by a normal distribution with $\mu=np$ and $\sigma^2=npq$, where $q=1-p$.</p>
<p>I think it is relatively intuitive to understand the Poisson approximation. Recall how do we get the poisson PMF from <strong>Lecture 11</strong>. We let $p = \cfrac{\lambda}{n}$ and make $n$ goes to infinity, we get:</p>
<p>$$
\begin{aligned}
&amp; \lim_{n \to \infty} \binom{n}{k}\left( \cfrac{\lambda}{n} \right)^k \left(1-\cfrac{\lambda}{n} \right)^{n-k} \\[10pt]
&amp;= \cfrac{\lambda^k}{k!} \lim_{n \to \infty} \cfrac{n(n-1)\cdots(n-k+1)}{n^k} \cdot \left(1-\cfrac{\lambda}{n} \right)^n \cdot \left(1-\cfrac{\lambda}{n} \right)^{-\lambda} \\[10pt]
&amp;= \cfrac{\lambda^k}{k!} \cdot \left( 1 \cdot 1 \cdot e^{-\lambda} \right)
\end{aligned}
$$</p>
<p>The bigger $n$ is and the smaller $p = \cfrac{\lambda}{n}$ is, the better the approximation. One question that still remains is: why does the central limit theorem failed here?</p>
<p>Recall from <strong>Lecture 15</strong> when we introduced the central limit theorem, we discussed &ldquo;how large is large enough&rdquo; in terms of sample sizes. It actually depends on the shape of the population distribution. When the population is normally distributed, $n=2$ is already large enough and its sampling distribution of the sample mean already follows a normal distribution. In our current situation, the population is heavily <em>heavily</em> skewed, with a $Ber(0.000001126088083)$. In this case, even 1 million is not large enough. Therefore, when dealing with proportions, &ldquo;large enough&rdquo; means $np \geqslant 10$. However, remember that the concept of &ldquo;success (1)&rdquo; or &ldquo;failure (0)&rdquo; is arbitrary. We also need $nq \geqslant 10$ at the same time, where $q=1-p$.</p>
<p>Now another question arises: why $np \geqslant 10$ AND $nq \geqslant 10$? Why not 5, 15, 20? Well &hellip; different people might have different thresholds. Some say 5, some say 10 like us, some say 15. Here, I&rsquo;m not able to provide a solid proof, but I can give a rough and intuitive reasoning.</p>
<p>From the above pictures, we see that one reason the normal distribution fails to approximate the binomial is that there is still a lot of data outside the interval $[0,n]$ where all data from the binomial is located. For a normal distribution, the data is located from $-\infty$ to $\infty$. From the <strong>empirical rule</strong>, we know that about $99.7\%$ of the data are located within the interval $[\mu-3\sigma, \mu+3\sigma]$. Intuitively, if this interval is contained within $[0,n]$, then we would not have the problem that a lot of the data in the normal distribution is outside the binomial&rsquo;s data range. Therefore, intuitively, we need:</p>
<p>$$0 &lt; \mu-3\sigma \textmd{ and } \mu+3\sigma &lt; n$$</p>
<p>We can replace the normal parameters with the binomial ones, starting from the first inequality:</p>
<p>$$
\begin{aligned}
0 &amp;&lt; \mu-3\sigma \\
0 &amp;&lt; np - 3\sqrt{npq} \\
np &amp;&gt; 3\sqrt{npq}\\
n^2p^2 &amp;&gt; 9npq \\
np &amp;&gt; 9q = 9(1-p) = 9 - 9p
\end{aligned}
$$</p>
<p>Since the maximum value of $9-9p$ is $9$, so we choose the threshold of $np \geqslant 10$. Now the second inequality:</p>
<p>$$
\begin{aligned}
\mu+3\sigma &amp;&lt; n \\
np + 3\sqrt{npq} &amp;&lt; n \\
3\sqrt{npq} &amp;&lt; n-np = n(1-p) = nq\\
9npq &amp;&lt; n^2 q^2 \\
nq &amp;&gt; 9p = 9(1-q) = 9 - 9q
\end{aligned}
$$</p>
<p>Similarly, the maximum value of $9-9q$ is $9$, so we choose the threshold of $nq \geqslant 10$.</p>
<p>In summary, when $n$ is large AND $np \geqslant 10$ AND $nq \geqslant 10$, we can use the normal distribution to approximate the binomial. When $n$ is large but $np$ or $nq$ is between 0 and 10, we can use the Poisson distribution for the approximation.</p>
<h2 id="the-wald-interval">The Wald Interval</h2>
<p>Now let&rsquo;s see how to construct the 95% confidence interval for the proportion. We use the same method as before, using the properties of the sampling distribution of the sample proportion:</p>
<p>$$
\begin{aligned}
\mathbb{P}\left(-1.96 \leqslant Z \leqslant 1.96 \right) = 0.95\\
\mathbb{P}\left(-1.96 \leqslant \cfrac{P-\mu_P}{\sigma_P} \leqslant 1.96 \right) = 0.95\\
\mathbb{P}\left(-1.96 \leqslant \cfrac{P-\pi}{\sqrt{\frac{\pi(1-\pi)}{n}}} \leqslant 1.96 \right) = 0.95\\
\mathbb{P}\left(P-1.96 \sqrt{\cfrac{\pi(1-\pi)}{n}}  \leqslant \pi \leqslant P + 1.96 \sqrt{\cfrac{\pi(1-\pi)}{n}} \right) = 0.95
\end{aligned}
$$</p>
<p>We manage to put the population proportion $\pi$ in the middle, but the lower and upper bounds are also dependent on $\pi$. This situation is different from the population mean and variance. What do we do here? Well, the easiest and the most intuitive way is basically replace the population proportion with the sample proportion:</p>
<p>$$\left[ p-1.96 \sqrt{\cfrac{p(1-p)}{n}}, p+1.96 \sqrt{\cfrac{p(1-p)}{n}} \right]$$</p>
<p>This is called the <strong>&ldquo;Wald interval&rdquo;</strong>.</p>
<p>Do we need to switch the $Z_{0.025}$ value $(\pm 1.96)$ to $\boldsymbol{t}$ values? Not really. The reason we use the $\boldsymbol{t}$-distribution for the mean is discussed. Whenever we get a sample mean, we still need the population standard deviation to get the confidence interval for the population mean. Since we do not know the population standard deviation, we need to use the sample data to independently<sup id="fnref:2"><a href="#fn:2" class="footnote-ref" role="doc-noteref">2</a></sup> calculate the sample standard deviation. When we use the sample standard deviation to replace the population standard deviation, we introduce some extra errors, so the distribution becomes the $\boldsymbol{t}$-distribution.</p>
<p>In the case of proportions, when we get a sample, the sample proportion $p$ is set. At the same time the standard deviation is also set, which is $\sqrt{npq}$, where $q=1-p$. We do not need to compute a standard deviation separately and there is no replacement. Therefore, the distribution is still normal.</p>
<p>There are other types of confidence intervals for proportions where you can check the <strong>References</strong>.</p>


<div class="admonition tip">
    <div class="title">References</div>
    <div class="content"><ul>
<li><a href="https://wellcomecollection.org/works/xmscu3g4">Ideas on protein synthesis (Oct. 1956)</a></li>
<li><a href="https://towardsdatascience.com/five-confidence-intervals-for-proportions-that-you-should-know-about-7ff5484c024f">Five Confidence Intervals for Proportions That You Should Know About</a></li>
</ul>
</div>
</div>
<div class="footnotes" role="doc-endnotes">
<hr>
<ol>
<li id="fn:1">
<p>Well &hellip; not always but most of the time. Some genes can be transcribed into functional non-coding RNAs without translating into proteins. However, whether we should call that a <strong>gene</strong>? I guess different people have different opinions.&#160;<a href="#fnref:1" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
<li id="fn:2">
<p>The mean and variance is independent. The proof is complicated and beyond the scope of this course.&#160;<a href="#fnref:2" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
</ol>
</div>

        </p>
    </div>

    <div class="prev-next">
        
    </div>
</div>

<aside class="post-toc">
    <nav id="toc">
        <nav id="TableOfContents">
  <ul>
    <li><a href="#a-simplified-version-of-gene-expression">A Simplified Version of Gene Expression</a></li>
    <li><a href="#conditions-of-the-normal-approximation">Conditions of The Normal Approximation</a></li>
    <li><a href="#the-wald-interval">The Wald Interval</a></li>
  </ul>
</nav>
    </nav>
</aside>



    

        </main><footer class="footer">
    <span>&copy; 2023 SUSTech BIO210 Biostatistics</span>
    <span>
        Made with &#10084;&#65039; using <a target="_blank" href="https://github.com/526avijitgupta/gokarna">Gokarna</a>
    </span>
</footer>
</body>
</html>
